lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -125,6 +155,19 @@ export class NativeInteractionClient extends BaseInteractionClient {
             provider,
             correlationId
         );
+        this.skus = setSKUs(this.nativeMessageHandler);
+    }
+
+    /**
+     * Adds SKUs to request extra query parameters
+     * @param request {NativeTokenRequest}
+     * @private
+     */
+    private addRequestSKUs(request: NativeTokenRequest) {
+        request.extraParameters = {
```

Comment:
Also, we should not modify input params. You should return a value instead

Reply:
This one modifies `validatedRequest` which is an extended copy of `request`. I do not think we need to add another spread operator here.

Reply:
We fixed a bug related to this pattern last week and I have a work item open to resolve remaining instances & turn on linting to catch in the future. We can discuss offline if you like.